legal-api/tests/unit/services/filings/validations/test_amalgamation_application.py

```diff
@@ -1620,6 +1616,7 @@ def mock_find_by_identifier(identifier):
         "legal_api.services.bootstrap.AccountService.get_account_by_affiliated_identifier",
         return_value={"orgs": [{"id": account_id}]} if test_status == "SUCCESS" else {},
     )
+    mocker.patch("legal_api.models.legal_entity.LegalEntity.validate_identifier", return_value=True)
```

Comment:
We need to patch `validate_identifier` here as it is called when the identifier is set for a LegalEntity. In these tests we use `mock_find_by_identifier`, which creates a new LegalEntity with potentially an expro identifier (which will fail the `validate_identifier` check).